src/internal/ClientImpl.ts

```diff
@@ -207,6 +207,11 @@ export class ClientImpl implements FluenceClient {
             },
         });
 
+        if (res.result === undefined) {
+            log.error('Call service unexpectedly returned undefined result, falling back to null');
```

Comment:
Can you plz add serviceId and fnName to the error?

Reply:
Great idea! Done